.github/actions/dump-ci-stats-to-gcp-metrics.cjs

```diff
@@ -0,0 +1,169 @@
+const Monitoring = require('@google-cloud/monitoring');
+
+const gcpCredentials = JSON.parse(process.env.GCP_CREDENTIALS);
+const monitoring = new Monitoring.MetricServiceClient({
+  projectId: gcpCredentials.project_id,
+  credentials: {
+    client_email: gcpCredentials.client_email,
+    private_key: gcpCredentials.private_key,
+  },
+});
+
+async function sendMetricsToGCP(metricType, metricValue, labels) {
+  const projectId = gcpCredentials.project_id;
+
+  const request = {
+    name: monitoring.projectPath(projectId),
+    timeSeries: [
+      {
+        metric: {
+          type: `custom.googleapis.com/github/${metricType}`,
+          labels: labels,
+        },
+        resource: {
+          type: 'global',
+          labels: {
+            project_id: projectId,
+          },
+        },
+        points: [
+          {
+            interval: {
+              endTime: {
+                seconds: Math.floor(Date.now() / 1000),
+              },
+            },
+            value: {
+              doubleValue: metricValue,
+            },
+          },
+        ],
+      },
+    ],
+  };
+  try {
+    await monitoring.createTimeSeries(request);
+    console.log(`Metric ${metricType} sent successfully.`);
+  } catch (error) {
+    console.error('Error sending metric:', error);
+  }
```

Comment:
Probably I'll remove this catch infuture as incase of error, the step should fail.
But sometimes timeseries issue on a single datapoint is fixed on rpc retry and it should not affect other metrics.
So for now catch makes sense